### ISSUE TITLE
Dark scheme use same color for vars and functions

### ISSUE THREAD

**Issue by Kendos-Kenlen** [mallowigi/material-theme-jetbrains-extended#30]
Hi,

In the dark color scheme, vars and functions calls use the same color (Ruby and JavaScript for example) while others don't.

Separate them would make the code easier to read.

Sincerely,

**Comment by mallowigi** [comment]
I don't get what you are saying. In what I see, functions are in blue while variables are in pure white.

Are you sure you don't have a copy of the bundled themes?

**Comment by nazrhyn** [comment]
**Darker**   
![image](https://cloud.githubusercontent.com/assets/1672405/26128932/2bc6d9d2-3a5c-11e7-9f32-623077cfd6fb.png)

**Default**   
![image](https://cloud.githubusercontent.com/assets/1672405/26128947/3b12f3ee-3a5c-11e7-8647-dbb5bcf81e69.png)

Those are using the built-in Material themes unaltered.

**Comment by Kendos-Kenlen** [comment]
Hi,

Sorry about JS, I did a mistake, it was my fault.

However, this is a screenshot of my Ruby code :

![screenshot_20170517_101348](https://cloud.githubusercontent.com/assets/2579741/26144858/cb1cf49c-3aea-11e7-9d27-76a615020c74.png)

I investigate a bit and I think the problem comes from the settings themselves. There is no "function call" property, only identifiers (which is used to highlight function calls). I reported this to JetBrains.
